### DISCUSSION TITLE
Perhaps RigTools for v130 and lower versions?

### DISCUSSION THREAD

**Discussion by C0D3Hermit** [3kh0/ext-remover#1182] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I wonder if this is possible!

**Comment by akabutnicer** [comment]
I doubt it. This exploit has been dragged through enough versions uncaught, there probably won't be an unpatch for this.

**Reply by C0D3Hermit** [reply]
damn

**Comment by OPcoolman** [comment]
If you disabled extensions with it before on a version that didn't patch it you still can

**Comment by Charlieis1nsan3** [comment]
this is not possible due to the new v130 update.

**Comment by Charlieis1nsan3** [comment]
but you could use br1ck unenroll, and use KVS, and then downgrade your Chromebook to a lower version.

**Reply by buddyted** [reply]
then could you re enroll it or would you be cooked

**Reply by no-name-6655** [reply]
You can re-enroll it but I don't know why you would want to do that

**Reply by Charlieis1nsan3** [reply]
> You can re-enroll it but I don't know why you would want to do that

You don't wanna give your Chromebook Unenrolled, then their gonna ask you  to fucking pay like 200$ for a fucking 10$ Chromebook lol.
